apps/wallet/src/ui/app/components/user-approve-container/UserApproveContainer.module.scss

```diff
@@ -3,7 +3,7 @@
 .container {
     display: flex;
     flex-flow: column nowrap;
-    height: 100%;
+    height: 90vh;
```

Comment:
How does this handle transaction requests that scroll?

Reply:
this only addresses the permission request height.

Reply:
done.